stdlib/crypto/src/main/ballerina/src/crypto/crypto.bal

```diff
@@ -408,13 +434,14 @@ function externEncryptAesEcb(byte[] input, byte[] key, handle padding) returns b
 } external;
 
 # Returns AES GCM encrypted value for the given data.
+# ```ballerina byte[]|crypto:Error cipherText = crypto:encryptAesGcm(cipherText, key, initVector); ```
 #
 # + input - The content to be encrypted
 # + key - Encryption key
 # + iv - Initialization vector
 # + padding - The padding
 # + tagSize - Tag size
-# + return - Encrypted data or `Error` if key is invalid
+# + return - Encrypted data or `crypto:Error` if key is invalid
```

Comment:
```suggestion
# + return - Encrypted data or `crypto:Error` if key is invalid
```
Please refer the template for the correct format.

Reply:
Corrected.